frontend/src/components/Boards/MyBoards/index.tsx

```diff
@@ -148,37 +170,45 @@ const MyBoards = React.memo<MyBoardsProps>(({ userId, isSuperAdmin }) => {
                       Last updated -{' '}
                       {date === currentDate ? `Today, ${formatedDate}` : formatedDate}
                     </Text>
-                    {/* to be used on the full version -> <Flex justify="end" css={{ width: "100%" }}>
+                    {/* to be used on the full version -> */}
+                    <Flex justify="end" css={{ width: '100%' }}>
                       <Flex
                         css={{
-                          position: "relative",
-                          zIndex: "30",
-                          "& svg": { size: "$16" },
+                          position: 'relative',
+                          zIndex: '30',
+                          '& svg': { size: '$16' },
                           right: 0,
-                          top: "-22px",
+                          top: '-22px',
```

Comment:
```suggestion
                          top: '$-22',
```

And define the token as a string e.g. `'-2': -22px'